### DISCUSSION TITLE
Sidebar in noVNC Window at Lovelace Dashboard

### DISCUSSION THREAD

**Discussion by Sirch80** [Poeschl/Hassio-Addons#326] [Q&A]
Hi all,
i wanted to integrate a VNC window in a Homeassistant Dashoard. I realized this with the webpage Card an it worked well.
The only issue is that there is the whole Homeassistant Sidebar in the card also. That means you have a little Version of Homeassistant in the Card which is overlaying the VNC screen when the cardsize is not as big.
![grafik](https://user-images.githubusercontent.com/80119112/181280122-3a7be4f7-790a-4f7d-85db-c5a6502b12ba.png)

How can i get rid of the side or headbar in the card?
Have anybody soled this issue.
Thanks Chris

**Comment by mouws** [comment]
A bit late, but I believe this is what you look for:
https://github.com/maykar/kiosk-mode
